lib/invoker/power/setup/osx_setup.rb

```diff
@@ -1,10 +1,39 @@
 module Invoker
   module Power
     class OsxSetup < Setup
-      RESOLVER_FILE = "/etc/resolver/dev"
-      RESOLVER_DIR = "/etc/resolver"
       FIREWALL_PLIST_FILE = "/Library/LaunchDaemons/com.codemancers.invoker.firewall.plist"
 
+      class << self
+        # @!group Helpers for use in tests
```

Comment:
Did we just add these for tests? If so - that seems backward. Also, `install` method in setup class is being called via an instance not via class method.

I specifically don't like class vs instance method confusion and hence you will notice all existing methods are instance methods except `Setup.install` and `Setup.uninstall`, but they themselves instantiate right installer classes and call instance on them.

What I mean is - lets not have this confusion about class & instance methods and let everything be instance methods. Test has to work with this design and this code shouldn't be modified so as tests can work.